iso15118/secc/states/iso15118_2_states.py

```diff
@@ -881,8 +881,12 @@ async def process_message(
             #      allowing the CSMS to attempt to retrieve the root certificate
             #      and construct the OCSP data itself.
             root_cert_path = self._mobility_operator_root_cert_path()
-            root_cert = load_cert(root_cert_path)
-            verify_certs(leaf_cert, sub_ca_certs, root_cert)
+            try:
+                root_cert = load_cert(root_cert_path)
+                verify_certs(leaf_cert, sub_ca_certs, root_cert)
+            except FileNotFoundError:
+                logger.warning(f"MO Root Cert can not be found {root_cert_path}")
```

Comment:
Teeny tiny change - wouldn't "cannot" sound better than "can not"? Also "TODO GitHub#94" in the lines above can be removed now?